Specs/NumberWithUnit/German/CurrencyModel.json

```diff
@@ -286,7 +286,7 @@
   ]
 }
 ,
-{//Age-Model currently does not work with ordinals
```

Comment:
Same comment as in the datetime specs.

Reply:
Fixed!
Python does not support JSON files with c-style comments so we added a new JSON property named **Comment**